.env.docker

```diff
@@ -68,3 +68,6 @@ LTI1P3_TOOL_JWKS_URL=http://demo_lti1p3_nginx/lti1p3/.well-known/jwks/toolSet.js
 BLACKFIRE_SERVER_ID=<your_backfire_id>
 BLACKFIRE_SERVER_TOKEN=<your_backfire_secret>
 ###< blackfire ###
+
+WEBHOOK_USERNAME=webhook
+WEBHOOK_PASSWORD=cCVMxdX8hfctDefk
```

Comment:
Same proposal about changing naming to `BASIC_AUTH_USERNAME` / `BASIC_AUTH_PASSWORD`

Reply:
Agreed and renamed